pkgs/applications/science/biology/sratoolkit/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv, lib
```

```suggestion
{ stdenv
, lib
```
This was just an error in a treewide change